### PR TITLE
[test] apply the new test style to bendctl

### PR DESCRIPTION
**Pull request by PsiACE**
Signed-off-by: Chojan Shang <psiace@outlook.com>

I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

apply the new test style to bendctl

## Changelog

- Build/Testing/CI
- Not for changelog (changelog entry is not required)

## Related Issues

Part of #1866

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: cli/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,10 @@ license = "Apache-2.0"
 publish = false
 edition = "2021"
 
+[lib]
+doctest = false
+test = false
```
Comment (PsiACE) [inline cli/Cargo.toml:12]: `doctest = false` avoid doc tests
`test = false` just compile once

**Comment by PsiACE** [comment]
Unless necessary, all tests will be moved to the corresponding `tests/it` directory.

File: cli/Cargo.toml (hunk 1 above)
Comment (PsiACE) [reply cli/Cargo.toml:12]: There will be two fewer packages to compile

**Comment by BohuTANG** [comment]
Cool.
After the test refined, how we measure the code coverage?

**Comment by PsiACE** [comment]
> Cool. After the test refined, how we measure the code coverage?

bendctl involves almost no code coverage, I will check this issue in other package improvements. 

Continue with the previous method for now. There may be some degree of reduction, but I don't think there will be much difference overall.

**Pull request merged**